docs/l10n.md

```diff
@@ -0,0 +1,82 @@
+# Localization of Git LFS
+
+Git LFS now has support for a localization framework using [Gotext](https://github.com/leonelquinteros/gotext), a Go library based around the popular gettext format.
+Localization is important since the majority of people on the planet don't speak English and people should be able to use software in the language they're most comfortable with.
+In addition, having access to localized software is a great way for people learning a language to improve their technical vocabulary.
+
+## Choosing What to Translate
+
+Here are some things that should be translated:
+
+* Status messages
+* Error messages
+* Help output
+* Generally, anything that the user sees in the normal course of operation
+
+Here are some things that should not be translated:
+
+* Trace output (e.g., calls to `tracerx.Printf`)
+* Strings which have a functional or protocol use (e.g., `Basic` and other authentication schemes, HTTP verbs)
+* Names of programs, commands, command-line options, or subcommands (although you _should_ translate their help output)
+* Personal names, names of businesses, email addresses, and other proper nouns (most of which we should not typically have in our codebase)
+* The names of Git LFS and Git themselves
+
+## Making Text Translatable
+
+The easiest way to make a string translatable is to use wrap the string and any formatting arguments it takes in a call to `tr.Tr.Get`.
+For example, you might write this:
+
+```
+Print(tr.Tr.Get("fetch: Fetching reference %s", ref.Name))
+```
+
+If you have a string which varies based on a number, use `tr.Tr.GetN`, provide the singular string, the plural string, the number upon which it varies, and then the arguments:
+
+```
+Print(tr.Tr.GetN(
+	"fetch: Fetching changes within %v day of %v",
+	"fetch: Fetching changes within %v days of %v",
+	fetchconf.FetchRecentCommitsDays,
+	fetchconf.FetchRecentCommitsDays,
+	refName,
+))
+```
+
+Here are some tips for making your code as easy to translate as possible:
+
+* Avoid creating strings out of several independent words.
+  For example, rather than taking a string "upload" or "download" and appending "ing objects" to it, write the strings out in full as "Uploading objects" and "Downloading objects".
+  Not all languages compose words and phrases in the same way, and using full sentences or phrases in translations makes it easier to translate.
+* If you have a string which varies based on number, use `tr.Tr.GetN`, which handles pluralization correctly.
+  In some languages, zero is singular; in others, there are more than two forms of a word (e.g., singular, dual, and plural) or words may not vary in number at all.
```

Comment:
I cannot definitively say if there are languages with singular zero as well as multiple plural forms, although I'm not personally aware of any, but I will rephrase as you suggested to avoid being accidentally wrong.